### PR TITLE
fix multi intent unhappy path in docs

### PR DESCRIPTION
**Pull request by Ghostvv**
**Proposed changes**:
- fix multi intent unhappy path in docs
- fixes https://github.com/RasaHQ/rasa/issues/6861

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by erohmensing** [comment]
Akela wins, gonna make it a story for less confusing 😬

**Comment by Ghostvv** [comment]
@erohmensing should I close the PR then?

**Review by erohmensing — APPROVED** [review]

**Comment by erohmensing** [comment]
@Ghostvv looks fine right?

File: docs/docs/forms.mdx [hunk 1]
```diff
@@ -251,18 +252,19 @@ rules:
...
 the form and reset the requested slot. An example story of such conversation could
 look as follows:
 
 ```yaml-rasa
-rules:
-- rule: Example of an unhappy path
-  condition:
-  # Condition that form is active.
-  - active_loop: restaurant_form
+- story: User interrupts the form and doesn't want to continue
```
Comment (Ghostvv) [inline docs/docs/forms.mdx:263]: do we need key `stories:` above

File: docs/docs/forms.mdx [hunk 2]
```diff
@@ -251,18 +252,19 @@ rules:
...
   steps:
+  - intent: request_restaurant
+  - action: restaurant_form
+  - active_loop: restaurant_form
```
Comment (Ghostvv) [inline docs/docs/forms.mdx:267]: the problem with this story is that it'll not work if there was `chitchat` before `stop`

**Comment by tmbo** [comment]
should this target 2.x?

File: docs/docs/forms.mdx (hunk 2 above)
Comment (erohmensing) [reply docs/docs/forms.mdx:267]: I think that's fine, here we're only trying to show what to do if the user wants to stop, not handle all cases

**Pull request merged**